cmd/helpMessages.go

```diff
@@ -323,6 +323,20 @@ Set the environment variable AZCOPY_SPA_CERT_PASSWORD to the certificate's passw
    Please treat /path/to/my/cert as a path to a PEM or PKCS12 file-- AzCopy does not reach into the system cert store to obtain your certificate.
    --certificate-path is mandatory when doing cert-based service principal auth.
 
+Log in using a Device:
```

Comment:
We shall get this part reviewed by Scott as well, as this is something visible to customer.

Reply:
lgtm